src/fragments/lib/storage/js/download.mdx

```diff
@@ -9,17 +9,20 @@ await Storage.get(key: string, config: {
   download?: boolean, // defaults to false
   expires?: number, // validity of the URL, in seconds. defaults to 900 (15 minutes)
   contentType?: string // set return content type, eg "text/html"
+  validateObjectExistence?: boolean // defaults to false
 });
 ```
 
-`Storage.get` returns a signed URL `string` to your file, if `download` is false, which is the default. You can use this to create a download link for people to click on. This is our recommended option. Note that this method **does not check if the file actually exists** as that would involve an extra API call.
+`Storage.get` returns a signed URL `string` to your file, if `download` is false, which is the default. You can use this to create a download link for people to click on. This is our recommended option. Note that this method **does not check if the file actually exists by default**. But you can check for the existence of file by setting `validateObjectExistence` to `true`.
```

Comment:
Suggested changes to this line with some changes to the existing wording, please double check me.

```suggestion
`Storage.get` returns a signed URL `string` to your file if `download` is false, which is the default. You can use this to create a download link for users to click on. Note that this method **does not check if the file actually exists by default**. You can check that the file exists when generating a signed URL by setting `validateObjectExistence` to `true`.
```